.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+Please help potential reviewers to understand this pull request and speed up 
+the process by answering the following questions, where appropriate.
+
+# Why was this pull request created?
+
+# How is this achieved?
+
+# How to test this?
```

Comment:
I'm happy to be proven wrong, but we didn't have resources to do a manual per-commit/PR validation.